snap/snapcraft.yaml

```diff
@@ -410,8 +410,8 @@ parts:
       - -usr/share/
   support-rulesengine:
     source: https://github.com/edgexfoundry/support-rulesengine.git
-    # the following commit == 0.5.0 version update
-    source-commit: "2e282f5"
+    # the following commit == 0.7.0 version update
+    source-commit: "3aca4a7bf9774ad07b9b74c7b05c45deb74412f2"
```

Comment:
Same as above (ie. for support-scheduler).

Reply:
Same as above, not 0.7.0 tag